.circleci/config.yml

```diff
@@ -17,7 +17,7 @@ jobs:
           name: Build docs to store
           command: |
             cd doc
-            make html
+            make html SPHINXOPTS='-W'
```

Comment:
Currently we require ['sphinx >=1.7'](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/4c5dc3dee1d8ea2fcace39e6967f0c39510a41ad/doc/doc-requirements.txt#L1). If we upgrade to sphinx 1.8 we could add the ` --keep-going` flag to show all errors instead of just the first one.

https://www.sphinx-doc.org/en/1.8/man/sphinx-build.html

Reply:
+1 to upgrade